test/controllers/api/v2/users_controller_test.rb

```diff
@@ -206,7 +206,7 @@ def user_one_as_anonymous_viewer
   test "#index should not show hidden users" do
     get :index, params: { :search => "login == #{users(:anonymous).login}" }
     results = ActiveSupport::JSON.decode(@response.body)
-    assert results['results'].empty?, results.inspect
+    assert_empty results['results'], results.inspect
```

Comment:
I wonder if the `results.inspect` at the end is still needed. Previously it was there to provide some sane context because the result was just a bool. Now `assert_empty` can provide some useful feedback because it has the actual object.
```suggestion
    assert_empty results['results']
```

This is a big reason why `assert_empty` is a good idea.